scripts/update-json-schema.sh

```diff
@@ -34,7 +34,12 @@ git config user.name "$GIT_USER"
 git config user.email "$GIT_EMAIL"
 git add "$OUTPUT_DIR"
 
+# commit description
+if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then
```

Comment:
@cdrage $TRAVIS_PULL_REQUEST is always going to be false since this only work on merge to master. I think it's safe simply remove the PR number altogether to fix #555

Reply:
Yeah, that'd be better